Source/EasyNetQ/Consumer/HandlerRunner.cs

```diff
@@ -139,10 +139,19 @@ private string BuildErrorMessage(ConsumerExecutionContext context, Exception exc
                    string.Format("Exception:\n{0}\n", exception);
         }
 
-        public void Dispose()
+        public virtual void Dispose()
```

Comment:
please remove the `virtual` keyword here, the `Dispose(bool)` method is the one that should be overridden by subclasses.